config/locales/en.yml

```diff
@@ -1307,6 +1307,8 @@ en:
             trainee_id:
               blank: Enter a trainee ID
               max_char_exceeded: Your entry must not exceed 100 characters
+              uniqueness: The trainee ID has already been used for another trainee
+              uniqueness_explanation: The ID ‘%{trainee_id}’ has already been used for %{short_name} (created %{created}). Choose a new one or check if this is a duplicate trainee record.
```

Comment:
can we add the `no-wrap` class to the trainee id, their name and the date created please?

Reply:
Added